doc/openapi/firecrest-api.yaml

```diff
@@ -1,9 +1,15 @@
+##
+##  Copyright (c) 2019-2021, ETH Zurich. All rights reserved.
+##
+##  Please, refer to the LICENSE file in the root directory.
+##  SPDX-License-Identifier: BSD-3-Clause
+##
 openapi: 3.0.0
 servers:
   - url: 'http://FIRECREST_URL'
   - url: 'https://FIRECREST_URL'
 info:
-  version: 1.0.0-RC1
+  version: 1.7.0
```

Comment:
Big bump! Tasks suggestion for the backlog regarding the spec: 
- Integrate API version check in CICD
- Remove duplication in both specs
- Consider adding a layer like connexion or fastapi to increase coverage, consistency and validation of declared/implemented schemas, params, return types, etc.

Reply:
The "bump" is given because in the other releases openapi version change was missing.
+1 to suggestions.